data/road_maps.json

```diff
@@ -1,10 +1,4 @@
 [
-    {
```

Comment:
this removes the miro board, we intend to make another board? Or put a screenshot of this on the website?

Reply:
The miro board card was replaced with an embed to the same board.